Configure

```diff
@@ -512,6 +513,8 @@ my @disable_cascades = (
     # or modules.
     "pic"               => [ "shared", "module" ],
 
+    "dynamic-engine"    => [ "fips" ],
```

Comment:
No.  If anything, it should be this:

``` perl
    "module"                => [ "fips" ],
```

Reply:
As an aside the "no-modules" option appears to be undocumented.

Reply:
Very good point.  I'll get on it.